### PR TITLE
[win32] extend UWP with capabilities and file associations

### PR DESCRIPTION
**Pull request by MartijnKaijser**
See title.
Need to check if we indeed play all these files or more....

### CODE REVIEW

**Comment by hudokkow** [comment]
https://github.com/xbmc/xbmc/blob/master/xbmc/settings/AdvancedSettings.cpp#L385-L386

**Comment by MartijnKaijser** [comment]
Update the manifest files. I've indented all the file types that were not on our list to the right side.
Question now really is do we actually still support all the ones in 
https://github.com/xbmc/xbmc/blob/master/xbmc/settings/AdvancedSettings.cpp#L385-L386
or are there obsolete ones in it?
Maybe we also can play the others but are never added to our list like the ones indented to the right.
@FernetMenta perhaps knows?

**Comment by FernetMenta** [comment]
The list of extensions shows all types of formats xbmc/kodi may have been capable to play. Since nobody does continuous testing on how good this works, we don't know.
IMO file associations are kind of recommendations to the user. We should not recommend something we don't know. Go for the formats we know that those are working good and ignore the rest.

File: project/Win32BuildSetup/AppxManifest.xml.in [hunk 1]
```diff
@@ -23,10 +23,181 @@
...
+              <uap:FileType>.rec</uap:FileType>
+              <uap:FileType>.rm</uap:FileType>
+              <uap:FileType>.rma</uap:FileType>
+              <uap:FileType>.rmt</uap:FileType>
+              <uap:FileType>.rmvb</uap:FileType>
+              <uap:FileType>.rsd</uap:FileType>
+              <uap:FileType>.rss</uap:FileType>
+              <uap:FileType>.pva</uap:FileType>
+              <uap:FileType>.qt</uap:FileType>
+              <uap:FileType>.sap</uap:FileType>
```
Comment (MartijnKaijser) [inline project/Win32BuildSetup/AppxManifest.xml.in]: @stefansaraev wasn't .sap support removed some time ago?

Comment (hudokkow) [inline project/Win32BuildSetup/AppxManifest.xml.in]: https://github.com/xbmc/xbmc/pull/8612

Comment (stefansaraev) [inline project/Win32BuildSetup/AppxManifest.xml.in]: it was sap:// protocol from removed from our vfs. no idea what .sap extension is.

Comment (MartijnKaijser) [inline project/Win32BuildSetup/AppxManifest.xml.in]: Seems to be some atari music file. 
So maybe we missed removing some extensions when splitting into music decoder Addons

Comment (FernetMenta) [inline project/Win32BuildSetup/AppxManifest.xml.in]: nope, ffmpeg has a sap demuxer

**Pull request merged**